recipes/plugins.rb

```diff
@@ -28,10 +28,6 @@
 
 reload_file = ::File.join(Chef::Config[:file_cache_path], 'reload-jenkins')
 
-file reload_file do
-  action :nothing
-end
-
```

Comment:
Why was this removed?

Reply:
This has the same name as the `file` resource in line 48. They both have different actions so in Chef 12 the resource has a `:delete` action.